src/Compiler/lex.fsl

```diff
@@ -1427,15 +1455,124 @@ and tripleQuoteString sargs skip = parse
 
  | eof
     { let (_buf, _fin, m, kind, args) = sargs
-      EOF (LexCont.String(args.ifdefStack, args.stringNest, LexerStringStyle.TripleQuote, kind, m)) }
+      EOF (LexCont.String(args.ifdefStack, args.stringNest, LexerStringStyle.TripleQuote, kind, args.delimLength, m)) }
 
  | surrogateChar surrogateChar // surrogate code points always come in pairs
  | _
     { let (buf, _fin, m, kind, args) = sargs
       addUnicodeString buf (lexeme lexbuf)
-      if not skip then STRING_TEXT (LexCont.String(args.ifdefStack, args.stringNest, LexerStringStyle.TripleQuote, kind, m))
+      if not skip then STRING_TEXT (LexCont.String(args.ifdefStack, args.stringNest, LexerStringStyle.TripleQuote, kind, args.delimLength, m))
       else tripleQuoteString sargs skip lexbuf }
 
+and extendedInterpolatedString sargs skip = parse
```

Comment:
Are languageFeature guards not needed inside of here?
As in, code can only reach this block if the feature is enabled?

Reply:
That's the idea, we only get here if feature is enabled.